From: Michael Biebl <biebl@debian.org>
Date: Tue, 8 Jul 2014 21:06:07 +0200
Subject: escape: move to rootbindir

The systemd-escape utility might be used during early boot (e.g. when
being triggered from udev rules), so move it to rootbindir to support
systems with a split /usr setup.

(cherry picked from commit 2945a452b084879c6885efe51aefea00c179aa5f)
---
 Makefile.am | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index c856d82..7210ab7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -337,7 +337,8 @@ rootbin_PROGRAMS = \
 	systemd-notify \
 	systemd-ask-password \
 	systemd-tty-ask-password-agent \
-	systemd-machine-id-setup
+	systemd-machine-id-setup \
+	systemd-escape
 
 bin_PROGRAMS = \
 	systemd-cgls \
@@ -347,8 +348,7 @@ bin_PROGRAMS = \
 	systemd-delta \
 	systemd-analyze \
 	systemd-run \
-	systemd-path \
-	systemd-escape
+	systemd-path
 
 dist_bin_SCRIPTS = \
 	src/kernel-install/kernel-install
